hosts/hosts.go

```diff
@@ -272,6 +272,16 @@ func buildCleanerConfig(host *Host, toCleanDirs []string, cleanerImage string) (
 
 func NodesToHosts(rkeNodes []v3.RKEConfigNode, nodeRole string) []*Host {
 	hostList := make([]*Host, 0)
+	// If nodeRole is not set, return all
```

Comment:
how exactly the role is not set, the validation fails if the role not set in the config

Reply:
if its not passed to the function, we return all hosts